### PR TITLE
Issue 84 update readme (resolves #84)

### LINKED ISSUE #84: Edit readme.md to document new changes

### PR DESCRIPTION
**Pull request by cket**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
+Toil is a massively scalable pipeline management system, written entirely in Python.
+Toil runs as easily on a laptop as it does on a physical cluster or in the cloud, thanks
```
Comment (hannes-ucsc) [inline README.rst]: bare-metal instead of physical?

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+Toil is robust, and designed to run in highly unreliable computing environments like
+Amazon's `Spot Market`_. Towards this goal, Toil does not rely on a distributed file system.
+Instead, Toil abstracts a pipeline's global storage as a JobStore that can be stored
+either locally or on AWS. The result of this abstraction is a robust system that can be
+resumed even after an unexpected shutdown of every node in the cluster that resulted in the
+loss of all local data.
+
+Writing a Toil script requires only a knowledge of basic Python. Toil 'Jobs' are the
+basic unit of work in a Toil workflow, can dynamically spawn children jobs that will
```
Comment (hannes-ucsc) [inline README.rst]: two times "basic"

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+==================
+
+This is only required if cloning from Git. To run the tests, ``cd`` into the toil root directory
+and run::
+
+    make test
+
+For developers, running::
+
+    make develop
```
Comment (hannes-ucsc) [inline README.rst]: I would list "make develop" first. Then "make test". And "Building & Testing" sort of implies "developer" to me.

**Comment by benedictpaten** [comment]
Are you planning to write an overview of the interface? I would be happy to pair write this with you.

**Comment by benedictpaten** [comment]
Even if we don't provide a complete manual, at least a quick start tutorial would be helpful. I would like to focus on documentation as part of the next sprint.

**Comment by cket** [comment]
I talked to Hannes about this, he was against putting interface information in the readme. I think the plan is to cover API documentation with the help of sphinx

**Comment by benedictpaten** [comment]
Okay, I'm fine with this, but we'll need to do it in the next milestone.

**Pull request merged**